### ISSUE TITLE
Option to preserve existing JULIA_NUM_THREADS setting

### ISSUE THREAD

**Issue by oschulz** [julia-vscode/julia-vscode#1096]
It would nice to have a value for `julia.NumThreads` (empty string or so?) that results in keeping a `$JULIA_NUM_THREADS` already set in the environment (e.g. before starting VS Code).

We usually have machine-dependent settings for `$JULIA_NUM_THREADS` (depending on number of cores and hyperthreading), so having a way to tell the Julia extension to keep her fingers off it would  be cool.

**Comment by hsm207** [comment]
Let's close this issue. It's been fixed in PR #1104 .
